.pipelines/singletenancy/overlay/overlay-e2e-step-template.yaml

```diff
@@ -91,7 +96,14 @@ steps:
       kubectl create configmap config-reconcile.yaml
       cd ../../../..
       kubectl get po -owide -A
-      sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(make cns-version) CNI_DROPGZ_VERSION=$(make cni-dropgz-version) INSTALL_CNS=true INSTALL_OVERLAY=true TEST_DROPGZ=${{ parameters.testDropgz }}
+      # Nightly does not build images per commit. Will use existing image. 
+      if [ "$CILIUM_VERSION_TAG" = "cilium-nightly-pipeline" ] 
+      then 
+          CNS=v1.4.44_hotfix DROPGZ=v0.0.4 && echo "Running nightly"
```

Comment:
Can we parametrize or make it use the latest cns/dropgz version. 
The concern I have is we need to update this file every time we make a new cns/dropgz release. OR test with some CNS/dropgz version

Reply:
When using the latest cns/dropgz version the images themselves are sometimes not published leading to invalid image pulls. Even in other pipelines the :latest tag is overwritten by other versions to prevent this. 

I do agree that updating this will require extra work. My solution is to build and push the images before we run this nightly pipeline, but now we are doubling up on work that exists in other pipelines.

Reply:
Would be nice to have a task related to it under https://msazure.visualstudio.com/One/_workitems/edit/17949633, just to keep a track and we don't forget about it.